spec/support/shared/unit/provider/file.rb

```diff
@@ -43,7 +43,7 @@ def windows_path
 def setup_normal_file
   [ resource_path, normalized_path, windows_path].each do |path|
     allow(File).to receive(:file?).with(path).and_return(true)
-    allow(File).to receive(:exists?).with(path).and_return(true)
+    allow(File).to receive(:exist?).with(path).and_return(true)
     allow(File).to receive(:exist?).with(path).and_return(true)
```

Comment:
whole bunch of duplicated :exist? calls after this PR

Reply:
Fixed